linalg/src/x86_64_fma.rs

```diff
@@ -11,9 +11,78 @@ mod intel;
 pub fn plug(ops: &mut Ops) {
     if is_x86_feature_detected!("fma") {
         ops.mmv_f32 = Box::new(|_, _| mmm::fma_mmm_f32_64x1::mmm());
+
+        ops.mmm_f32 = Box::new(|_, _, n| {
+            if n.is_none() {
+                return mmm::fma_mmm_f32_16x6::mmm();
+            }
+
+            let n = n.unwrap();
+
+            match n {
+                1 => unreachable!("should've been mmv"),
+                2 => return mmm::fma_mmm_f32_40x2::mmm(),
+                3 => return mmm::fma_mmm_f32_32x3::mmm(),
+                4 => return mmm::fma_mmm_f32_24x4::mmm(),
+                5 => return mmm::fma_mmm_f32_16x5::mmm(),
+                6 => return mmm::fma_mmm_f32_16x6::mmm(),
+                8 => return mmm::fma_mmm_f32_8x8::mmm(),
```

Comment:
Mmmm... ok. so high level view of multiplier keep the old kernel naming convention, actually. So I guess you just introduced the new notation as a kernel implementation internal after all...

Reply:
Yepp! My plan was to do something smart in the build scripts to sort this out and wrote all templates on the 2x6, 2x5 etc patterns. But when dropping AVX-512 from this PR I renamed them to this scheme directly to avoid confusion.